jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,525 @@
+---
+layout: classic-docs
+title: "CircleCI API Developer's Guide"
+short-title: "Developer's Guide"
+description: "API cookbook for internal and external CircleCI developers"
+categories: [getting-started]
+order: 1
+---
+
+This *API Developer's Guide* was written to assist developers in quickly and easily making API calls to CircleCI services to return detailed information about users, pipelines, projects and workflows.
+
+* TOC
+{:toc}
+
+# API Overview
+
+The CircleCI platform provides a powerful API that enables users to retrieve detailed information about users, jobs, workflows and pipelines.
```

Comment:
~\~ Is there a way to rephrase this without including the word "powerful"? The API can also be used to interact with CircleCI, e.g. triggering pipelines, updating settings etc.

Reply:
fixed in document